colour/models/rgb/rgb_colourspace.py

```diff
@@ -345,6 +490,100 @@ def decoding_cctf(self, value):
                     'decoding_cctf', value))
         self._decoding_cctf = value
 
+    @property
+    def use_derived_RGB_to_XYZ_matrix(self):
+        """
+        Property for **self._use_derived_RGB_to_XYZ_matrix** private attribute.
+
+        Returns
+        -------
+        bool
+            self._use_derived_RGB_to_XYZ_matrix.
+        """
+
+        return self._use_derived_RGB_to_XYZ_matrix
+
+    @use_derived_RGB_to_XYZ_matrix.setter
+    def use_derived_RGB_to_XYZ_matrix(self, value):
+        """
+        Setter for **self._use_derived_RGB_to_XYZ_matrix** private attribute.
+
+        Parameters
+        ----------
+        value : bool
+            Attribute value.
+        """
+
+        assert isinstance(value, bool), (
```

Comment:
Thought I had replied to this, may not have pushed the *Add single comment* button.
One issue with this test is that you can pass something like `bool('False')`, `bool('0')`, `bool('Off')`, etc... and still get `True`.

Overall I think I want to rework all those properties checks in the API, *Bokeh* seems to be doing something smart which might be clean, I need to investigate that.